src/itemChoice/__snapshots__/index.unit.tsx.snap

```diff
@@ -47,7 +47,7 @@ exports[`ItemChoice Should display a Loader when the component is in loading sta
 }
 
 .c2.kirk-text-button {
-  color: #054752;
+  color: #00AFF5;
```

Comment:
Normal change as I updated `TextDisplayType.BUTTON` with `color.blue` instead of `color.midnightGreen`